app/views/brexit_checker/_action_audiences.html.erb

```diff
@@ -9,7 +9,8 @@
             </h2>
           </header>
           <% audience[:groups].each.with_index do |group, group_index| %>
-            <section class="brexit-checker-actions__group">
+            <% ecommerce_list = group[:heading].nil? ? audience[:heading] : group[:heading] %>
+            <section class="brexit-checker-actions__group" data-analytics-ecommerce data-ecommerce-start-index="1" data-list-title="Brexit checker results: <%= ecommerce_list %>" data-search-query>
```

Comment:
interesting what does that `data-search-query` do?

Reply:
@huwd It's actually designed for ecommerce tracking on our search pages, and when used there it stores the query that the user has typed in ([see code here](https://github.com/alphagov/static/blob/master/app/assets/javascripts/analytics/ecommerce.js#L13)). Annoyingly the way the code has been written means that it is required when setting up ecommerce tracking, even outside of search pages. We could change this, but it'd be a separate piece of work.